NAMESPACE

```diff
@@ -9,4 +9,6 @@ export(getDesignMatrix)
 export(linColumnFinder)
 export(miRNAPathwayEnrichment)
 import(dplyr)
+import(igraph)
+import(limma)
```

Comment:
Please run `devtools::document()` to update the `NAMESPACE` file after making changes to the docs.